docs/src/HomFreezingPlots.jl

```diff
@@ -0,0 +1,138 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+
+const CMT = CM.CommonTypes
+const CMO = CM.Common
+const CMI_het = CM.HetIceNucleation
+const CMI_hom = CM.HomIceNucleation
+const CMP = CM.Parameters
+
+include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
+FT = Float64
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+const prs = cloud_microphysics_parameters(toml_dict)
+thermo_params = CMP.thermodynamics_params(prs)
+
+# Initializing
+temp = collect(229.0:0.25:239.5)   # air temperature
+#x_sulph = 0.031                    # wt% sulphuric acid in droplets
+#x_sulph = 0.04
+x_sulph = 0.065
+Delta_a = Vector{Float64}(undef, length(temp))
+J = Vector{Float64}(undef, length(temp))
+a_w = Vector{Float64}(undef, length(temp))
+
+it = 1
+for T in temp
```

Comment:
Could I ask you to update this one to the same way of plotting as the water activity ones - i.e. without the loop and the global it?